src/soap/serializers/value.js

```diff
@@ -40,21 +47,41 @@ export function valueInfoType({
 }) {
   if (!(datapoint && typeof datapoint === 'object')) return ''
 
-  const censorCode = 'nc'
+  const censorCode = datapoint.d && datapoint.d.CensorCode
   const dateTime = new Date(datapoint.lt).toISOString().replace('.000Z', '')
-  const timeOffset = '-05:00'
   const dateTimeUTC = new Date(datapoint.t).toISOString().replace('.000Z', '')
   const value = datapoint.v
-
-  return `<value ${censorCode ? `censorCode="${encodeXML(censorCode)}"` : ''} ${
-    dateTime ? `dateTime="${dateTime}"` : ''
-  } ${timeOffset ? `timeOffset="${encodeXML(timeOffset)}"` : ''} ${
-    dateTimeUTC ? `dateTimeUTC="${dateTimeUTC}"` : ''
-  } ${methodId ? `methodCode="${encodeXML(methodId)}"` : ''} ${
-    sourceID ? `sourceCode="${encodeXML(sourceID)}"` : ''
-  } ${
+  const utcTimeOffset = datapoint.d && datapoint.d.UTCOffset
+
+  /**
+   *
+   * @param {string} n
+   * @returns -6 => -06 || 6 => 06 || 10 => 10 || 0 => 00
+   */
+  const twoDigitNumber = n => {
```

Comment:
I would just create and export a serializer called `timeOffset` at the top of the file that takes a number `offset` and serializes it to the formatted string. Basically combine the logic of twoDigitNumber with the other logic. Keeps it all in one place and can be reused (or have a separate test case some day if we desire).